provider/manual/environ_test.go

```diff
@@ -147,11 +147,6 @@ exit 0
 	}
 }
 
-func (s *environSuite) TestSupportsNetworking(c *gc.C) {
-	_, ok := environs.SupportsNetworking(s.env)
-	c.Assert(ok, jc.IsFalse)
```

Comment:
Rather than get rid of the test, can we just do `js.IsTrue`.

Reply:
It's represented in the new `environNetworkSuite`.

Reply:
ah